### ISSUE TITLE
Issue: Missing column when creating a view table

### ISSUE THREAD

**Issue by andres-vizcaino** [pocketbase/pocketbase#2604]
I encountered an issue when creating a view table using the following query:

```sql
SELECT p.id AS id, p.name AS product_name,
       pv.price AS venezuela_price, pv.id AS venezuela_price_id,
       pm.price AS open_market_price, pm.id AS open_market_price_id
FROM products p
JOIN prices pv ON p.id = pv.product
JOIN countries cv ON pv.country = cv.id AND cv.name = 'Venezuela'
JOIN prices pm ON p.id = pm.product
JOIN countries cm ON pm.country = cm.id AND cm.name = 'Open market'
```

The problem is that the column named "venezuela_price" is not being created when executing the above query. However, if I remove the "open_market_price" from the query, the "venezuela_price" column is created successfully.

![image](https://github.com/pocketbase/pocketbase/assets/94866827/2e588bba-194c-4ba5-b4e1-1164311b311b)

I am using version 0.14.0 of the library. I have checked the release notes but could not find any information regarding changes related to this issue. I would appreciate any assistance or clarification regarding this matter.

### Expected Behavior:
The column "venezuela_price" should be created when executing the query to create the view table.

### Actual Behavior:
The "venezuela_price" column is not created when executing the query.

### Environment:
version: 0.14.0

Thank you for your attention to this matter. I look forward to any insights or comments regarding this issue.

**Comment by ganigeorgiev** [comment]
There were several fixes related to the views in v0.15+ releases (including incorrectly resolving multiple aliased fields originating from the same field source).

Please try to upgrade and see if it will fix your issue (you'll have to resubmit the view query).

If you are still experiencing the issue after the update, please attach your collections configuration as JSON from the "Admin UI > Settings > Export collections".

**Comment by andres-vizcaino** [comment]
Working with v16.3. thank you very much.
